frontend/src/hooks/pims-api/useApi.test.ts

```diff
@@ -39,18 +39,19 @@ describe('useApi testing suite', () => {
     });
   });
 
-  it('useApi uses custom axios - failure', () => {
+  it('useApi uses custom axios - failure', async () => {
     mockAxios.onGet('failure').reply(400, 'failure');
-
-    renderHook(async () => {
-      const api = useAxiosApi();
-      try {
-        await api.get('failure');
-      } catch (error) {
-        expect(error.response.status).toBe(400);
-        expect(error.response.data).toBe('failure');
-        expect(mockAxios.history.get).toHaveLength(1);
-      }
+    let api = {} as any;
+    renderHook(() => {
+      api = useAxiosApi();
     });
```

Comment:
This probably works okay but I think best practice is to use the return value from `renderHook`:

```ts
const { result } = renderHook(useAxiosApi); // no need to create a callback function if hook doesn't take any params
const api = result.current;

// do whatever with api
```